src/installationmanager.h

```diff
@@ -279,6 +285,9 @@ private slots:
 
 private:
 
+  // The plugin container, mostly to check if installer are enabled or not:
```

Comment:
Not a fan of putting `:` at the end of comments, happens in some places but not all.

Reply:
I tend to follow what's in place, but when there is nothing I tend to use my own style with the `:`. I've replaced the `:` by `.`.